scripts/zones/Mount_Zhayolm/mobs/Ignamoth.lua

```diff
@@ -0,0 +1,34 @@
+-----------------------------------
+-- Area: Mount Zhayolm
+--  MOB: Ignamoth
+-----------------------------------
+require("scripts/globals/status")
+-----------------------------------
+
+function onMobSpawn(mob)
+    mob:addMod(dsp.mod.DOUBLE_ATTACK, 50)
+    mob:addMod(dsp.mod.REGAIN, 50)
```

Comment:
> regain and double attack is made up %
it for sure has regain, and higher DA than its job would provide?

These are pretty obvious in game, so we can test easily

in general do not make things up to give it "challenge". no custom.

Reply:
its war/war in your mob_pools entry so he'd already have DA